lib/luminork-server/src/service/v1/schemas/create_action.rs

```diff
@@ -104,6 +105,18 @@ pub async fn create_variant_action(
     )
     .await?;
 
+    // Publish an event for the authoring UI to have ready.
+    let summary = func
```

Comment:
I really think we should try and push these back to the DAL so we have the same logic as we do in SDF - I don't think building a frointend type here is the right thing to do

Reply:
I started that way. It turned out a bit funky.

I tried to make an authoring client func to publish the event, but in turn that will always generate a summary. That's not a bad abstraction, but I think this will require a longer pull to move all that logic into the dal. The good news is that I think you can do it. I think it'll just be bit a bit churn-y to pull the whole thread in this PR.

Let try it again and I'll re-request.

Reply:
@stack72 thoughts on the new shape?